packages/components/RoadmapSection.tsx

```diff
@@ -34,7 +33,12 @@ export default function RoadmapSection() {
         my="2rem"
       >
         {cardList.map((cardProps) => (
-          <CardItem key={cardProps.title} color={cardProps.color} link={cardProps.link} title={t(cardProps.title)} />
+          <CardItem
```

Comment:
Same issue of https://github.com/podcodar/webapp/pull/201/files#r1804012331

![image](https://github.com/user-attachments/assets/49b0e487-a29e-4054-add9-8f53ba4ba408)